### ISSUE TITLE
v0.7.0 does not yet support JupyterLab 3

### ISSUE THREAD

**Issue by jamesmyatt** [QuantStack/jupyterlab-drawio#70]
The current version of this extension cannot be installed with Jupyterlab 3

```bat
call conda create --name jl3-test --channel conda-forge --yes jupyterlab=3 python=3.8 nodejs=14
call conda activate jl3-test
jupyter labextension install jupyterlab-drawio
```

```
An error occured.
ValueError: The extension "jupyterlab-drawio" does not yet support the current version of JupyterLab.


Conflicting Dependencies:
JupyterLab              Extension      Package
>=3.0.0 <3.1.0          >=2.0.0 <3.0.0 @jupyterlab/application
>=3.0.0 <3.1.0          >=2.0.0 <3.0.0 @jupyterlab/apputils
>=5.0.0 <5.1.0          >=4.0.0 <5.0.0 @jupyterlab/coreutils
>=3.0.0 <3.1.0          >=2.0.0 <3.0.0 @jupyterlab/filebrowser
>=3.0.0 <3.1.0          >=2.0.0 <3.0.0 @jupyterlab/launcher
>=3.0.0 <3.1.0          >=2.0.0 <3.0.0 @jupyterlab/mainmenu
```

**Comment by Starslayerx** [comment]
Got same issue, need jupyter 3.03 support.

**Comment by marimeireles** [comment]
Hey folks, I'm working on it :)

**Comment by marimeireles** [comment]
In the meanwhile please fix the download version of jupyterlab to 2.

**Comment by krassowski** [comment]
@marimeireles did you consider also incorporating the embedded mode from a PR by @bollwyvl for the next version? It would be awesome!

**Comment by bollwyvl** [comment]
Hm... i wouldn't consider/read that PR, as such, anymore... I'll probably be doing some work on the fork mentioned there (which has wildly diverged, purely to support needs of personal projects), and would certainly [prioritize lab 3](https://github.com/deathbeds/jupyterlab-drawio/issues/11) as one of the next "gotta-have" features.

**Comment by marimeireles** [comment]
Hey @krassowski not really, I just jumped in here for this jlab3 update :)
I think Wolf is the designated person for this kind of decision.

**Comment by marimeireles** [comment]
But, seems like awesome work @bollwyvl :)
Thanks!

**Comment by bollwyvl** [comment]
Well, that PR is _in_ the history of fork, somewhere, but the ensuing couple PRs landed some heavy features along the way. There's very little code left from this repo, and the packaging approach changed even more (separate package for drawio altogether). 

I guess it comes down to who "claims" the `jupyterlab_drawio` or `jupyter_drawio` PyPI package. I'm fine to leave it to this project, and can do a full rename.

**Comment by wolfv** [comment]
Just released 0.8.0 that fixes jlab 3.0 compat.
